cmd/jujud/util/util.go

```diff
@@ -115,18 +115,16 @@ func AgentDone(logger loggo.Logger, err error) error {
 	return err
 }
 
-// Pinger provides a type that knows how to ping.
-type Pinger interface {
-
-	// Ping pings something.
-	Ping() error
+// Brokener provides a type that exposes a "broken" channel.
+type Brokener interface {
```

Comment:
nooooooooooooooooooooooooooooooooooooooo :-( :-( :-( :-(

There *has* to be a better name